examples/scf/15-initial_guess.py

```diff
@@ -86,13 +86,35 @@
 mf.kernel()
 
 #
-# Superposition of atomic potentials can be used as initial guess for DFT
+# Superposition of atomic potentials (SAP) can be used as initial guess for DFT
 # methods.
 #
 mf = mol.RKS().set(xc='b3lyp')
 mf.init_guess = 'vsap'
 mf.kernel()
 
+#
+# Gaussian fitted version of SAP can be used with
+# any method.
+#
+mf = scf.RHF(mol)
+mf.init_guess = 'sap'
+mf.kernel()
+#
+# The SAP fit basis can be changed using the SCF object attribute sap_basis.
+# sap_basis accepts either python dictionary (basis set in internal
+# format) or filename/pathname. If BSE API is installed with pip, 
+# the implementation will also look through the BSE catalog for basis sets.
+#
+mf.sap_basis = 'sapgraspsmall'
+mf.kernel() # Will use PySCF SAP_ALIAS and find it
+# in the local files
+mf.sap_basis = 'sap_helfem_large'
+mf.kernel() # Will be found from BSE if installed
+mf.sap_basis = gto.basis.load('sap_helfem_large', 'Ne')
```

Comment:
This doesn't make sense? This should be a dict containing basis sets for every element! Also, the molecule has C not Ne...